README.md

```diff
@@ -80,7 +80,7 @@ Possible options are listed in the following table. More details are [in the cod
 | `layer` | [`ILayer`](http://leafletjs.com/reference.html#ilayer)  | The layer that the user's location should be drawn on. | a new layer |
 | `setView` | `boolean`  or `string`  | Set the map view (zoom and pan) to the user's location as it updates. Options are `false`, `'once'`, `'always'`, `'untilPan'`, or `'untilPanOrZoom'` | `'untilPanOrZoom'` |
 | `flyTo` | `boolean` | Smooth pan and zoom to the location of the marker. Only works in Leaflet 1.0+. | `false` |
-| `keepCurrentZoomLevel` | `boolean`  | Only pan when setting the view. | `false` |
+| `keepCurrentZoomLevel` | `boolean` or `Array` | Only pan when setting the view. Set to True, or a zoom range like [13,18] which will only zoom the map when outside the rage. Allows keeping the current view at these scales | `false` |
```

Comment:
Please correct the grammar of this sentence. Also, end with a `.`.

Reply:
Not sure what the grammer issue is, but have just spotted the rage=>range typo. I'll try fixing this - I think as a new PR.

Reply:
Something like this (it wasn't clear what the which clause refers to):

```suggestion
| `keepCurrentZoomLevel` | `boolean` or `Array` | Only pan when setting the view. Setting a zoom range such as `[13,18]` will only zoom the map is the user is zoomed in the given range. | `false` |
```

Reply:
just saw reply. I've updated the wording, try to keep clearer. It only zooms when outside the range. It keeps the zoom when within the range! ie keep zoom (if True, or within set range) https://github.com/barryhunter/leaflet-locatecontrol/commit/1eaee571533969af8c888b7b7d745490e4bfb255 
not clear if that commit, is feeding back to pull request. if not will make a new one, after resolving other comment)